keywords/LiteServiOS.py

```
@@ -137,6 +144,17 @@ def install(self):
                 time.sleep(1)
                 continue
 
+        # Get the device ID
+        list_output = subprocess.Popen(["xcrun", "simctl", "list"], stdout=subprocess.PIPE)
+        output = subprocess.check_output(('grep', 'Booted'), stdin=list_output.stdout)
+
+        for line in output.splitlines():
+            if "Phone" in line:
+                self.device_id = re.sub(' +', ' ', line).strip()
+                self.device_id = self.device_id.split(" ")[4]
+                self.device_id = self.device_id.strip('(')
+                self.device_id = self.device_id.strip(')')
```

Should we raise an exception is the device id is still `None` at this point?